app/assets/stylesheets/sufia/admin.scss

```diff
@@ -33,6 +33,10 @@ body {
   background-color: $admin-sidebar-background-color;
 }
 
+fieldset {
```

Comment:
@jcoyne So I should remove the `width: 100%` and try `float: left` instead? K, I can give that a shot.